test/integration/index.js

```diff
@@ -0,0 +1,94 @@
+'use strict'
+
+const {test} = require('tap')
+const config = require('../../config.js')
+const nock = require('nock')
+const pkg = require('../../package.json')
+const URL = 'https://ingest.pipeline.com'
+const KEY = 'abc123'
+process.env.PIPELINE_KEY = KEY
+process.env.PIPELINE_URL = URL
+
+// Internal Modules
+const index = require('../../index.js')
+
+const rawEvent = {
+  awslogs: {
+    // eslint-disable-next-line max-len
+    data: 'H4sIAAAAAAAAEzWQQW+DMAyF/wrKmaEkJCbhhjbWCzuBtMNUVSmkNBIQRMKqqep/X6Cb5Ivfs58++45G7ZzqdfMza5Sjt6IpTh9lXReHEsXI3ia9BJnQlHHIhMSEBnmw/WGx6xwcp8Z50M9uN2q/aDUGx2vn/5oYufXs2sXM3tjp3QxeLw7lX6hS47lTz6lTO9i1uynfXkOMe5lsp9Fxzyy/9eS3hTsyXYhOGVCaEsBSgsyEYBkGzrDMAIMQlAq+gQIQSjFhBFgqJOUMAog34WAfoFFOOM8kA0Y5SSH+f0SIb67GRaHq/baosn1UmUlHF7tErxvk5wa56b2Z+iRJ0OP4+AWj9ITzSgEAAA=='
+  }
+}
+
+const eventData = {
```

Comment:
`eventData` is unused, and eslint _is_ catching it, but it's not failing the build.

need to make sure `npm run lint` is executed as part of the checks in CI, before `npm run test`